customTests/azure_nccl_allreduce_ib_loopback.nhc

```diff
@@ -45,10 +42,7 @@ function check_nccl_allreduce_ib_loopback() {
          then
             IFS=$' \t\n'
             nccl_allreduce_ib_loopback_out_line=( ${nccl_allreduce_ib_loopback_out_lines[$i]} )
-            log "loopback: $nccl_allreduce_ib_loopback_out_line"
             avg_bus_bw=${nccl_allreduce_ib_loopback_out_line[5]}
-            log "abg bus bw: $avg_bus_bw"
-            dbg "Measured Avg NCCL allreduce ib loopback bus BW $avg_bus_bw GB/s (expected >=$EXP_NCCL_ALLREDUCE_IB_LOOPBACK_BW GB/s)"
```

Comment:
Yea. The nhc logs output when there is an error. It will also log output when we explicitly tell it to, just like above. In this case it is logging during a passing condition which isn't a big issue but if another test fails the output of that error is hard to find because of these log statements. So long story short we want to minimize logging unless we have a fail case.